src/scss/components/alert/index.scss

```diff
@@ -1,35 +1,35 @@
 hx-alert {
-  background-color: $gray-400;
-  color: $gray-1000;
+  background-color: var(--hxAlert, $gray-400);
+  color: var(--hxColor, $gray-1000);
```

Comment:
All CSS variables should be scoped to `hxAlert`:

```css
color: var(--hxAlert-color, $gray-1000);
```

Reply:
Changes have been done,Please review